hm-post-repeat.php

```diff
@@ -588,3 +622,72 @@ function get_available_repeat_types() {
 function is_allowed_repeat_type( $repeat_type ) {
 	return in_array( $repeat_type, array_keys( get_available_repeat_types() ) );
 }
+
+/**
+ * Add a row link to view all Repeat posts for the current Repeating post.
+ *
+ * The link text is `# repeat posts` where # is the number of corresponding Repeat posts.
+ *
+ * The filter is evaluated only for non-hierarchical post types.
+ *
+ * @param array   $actions An array of row action links.
+ * @param WP_Post $post    The post object to display row links for.
+ *
+ * @return array An array of row action links with added link to view Repeat posts and there count.
+ */
+function admin_table_row_actions_view_repeat_posts( $actions, $post ) {
+
+	if ( ! is_repeating_post( $post->ID ) ) {
+		return $actions;
+	}
+
+	// Get associated Repeat posts for the current Repeating post.
+	$url_args = array(
+		'post_type'      => get_current_screen()->post_type,
+		'hm-post-repeat' => 'repeat',
+		'post_parent'    => $post->ID,
+	);
+
+	// TODO: I think this is an expensive thing to do per each post. Maybe we can store Repeat post count in meta data?
+	$repeat_posts_query = new \WP_Query( get_repeat_type_query_params( 'repeat', $post->ID ) );
+	$repeat_posts_count = $repeat_posts_query->post_count;
```

Comment:
@willmot is there a better approach to store the count of repeat posts to make it quicker? Maybe a maintained meta data?

Reply:
TBH I'd be tempted to just skip showing the number in that list, I'm not sure it's worth the complexity. Especially as you'll see the count one you click the link.

Reply:
Actually, I've chatted to @johnbillion and he suggested to run a query with all parents IDs that currently on the page, that sounds like a good solution - I'm working at it atm.

When you click to see a specific post's Repeat posts, the overall number in the view link doesn't not change! It shows count for all Repeating and Repeat posts overall. This is similar to core WP, when you click a specific author (not yourself) the view link looses the bold focus, but the count stays the same. Hope that makes sense.

Reply:
Ah good point, and nice idea!